pkg/conversion/converter.go

```diff
@@ -40,13 +40,20 @@ type NameFunc func(t reflect.Type) string
 
 var DefaultNameFunc = func(t reflect.Type) string { return t.Name() }
 
+type ConversionFunc func(a, b interface{}, scope Scope) (bool, error)
```

Comment:
nit: Can you please rename it to GenericConversionFunc?

Having ConversionFunc and ConversionFuncs (which is completely different) is slightly misleading.